### ISSUE TITLE
Add a year filter to HOME screen

### ISSUE THREAD

**Issue by aaronjplatt** [aaronjplatt/tester#7]
RIO-17: Needs to be year and month filter

**Comment by aaronjplatt** [comment]
Original reporter: aaronjplatt

**Comment by aaronjplatt** [comment]
aaronjplatt:  1hr to add the filter
1hr to add year to the underlying pre-calculated tables
1hr to rebuild

**Comment by aaronjplatt** [comment]
aaronjplatt:  Added a year filter
Added a month year filter
Added a company filter
Added a business unit filter

I had to completely re-write the underlying tables to make this work.

Adding the initial quick filters wasn't quick. The data sources are all different, so to use one filter across them all required the use of tableau parameters. I built all the menus using parameters, but then realsied that tableau does not support context sensitive drop down menus with parameters. That means that the business unit list will always display all the business units, no matter what you select from COMPANY. Not ideal.

I rebuilt the underlying tables to put them all into one super summary table. This enables us to have one filter across all four metrics, and be context sensitive.
